connectors/2.0/apidocs/allclasses-frame.html

```diff
@@ -2,10 +2,10 @@
 <!-- NewPage -->
 <html lang="en">
 <head>
-<!-- Generated by javadoc (1.8.0_252) on Fri Jun 12 15:10:14 BST 2020 -->
+<!-- Generated by javadoc (1.8.0_265) on Sun Sep 27 09:10:34 BST 2020 -->
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-<title>All Classes (jakarta.resource API 2.0.0-RC2 API)</title>
-<meta name="date" content="2020-06-12">
+<title>All Classes (jakarta.resource API 2.0.0-SNAPSHOT API)</title>
```

Comment:
The version displayed on the Javadoc pages says 2.0.0-SNAPSHOT.  It should indicate the final 2.0.0 version (no SNAPSHOT).

Reply:
Replaced with contents of staged javadoc jar